tests/baselines/reference/narrowingPastLastAssignment.errors.txt

```diff
@@ -160,4 +161,14 @@ narrowingPastLastAssignment.ts(90,20): error TS7005: Variable 'x' implicitly has
         }
         values.forEach(v => foo.push(v));
     }
-    
\ No newline at end of file
+    
+    function f13() {
+        var foo: string | undefined;

```

Comment:
I guess the subtlety here is that this is `var` and I missed that. So if you add a comment to the original test like

```
// Test for captured 'var' declaration (as opposed to parameters, let, const).
```

that might be helpful.

Reply:
Comment added